test/parallel/test-whatwg-encoding-global.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const { TextEncoder, TextDecoder } = require('util');
+
+assert.deepStrictEqual(
+  Object.getOwnPropertyDescriptor(global, 'TextEncoder'),
+  {
+    value: TextEncoder,
+    writable: true,
+    configurable: true,
+    enumerable: false
+  }
+);
+
+assert.deepStrictEqual(
+  Object.getOwnPropertyDescriptor(global, 'TextDecoder'),
+  {
+    value: TextDecoder,
+    writable: true,
+    configurable: true,
+    enumerable: false
+  }
+);
```

Comment:
This needs a new line at the end

Reply:
@AyushG3112 Done: https://github.com/nodejs/node/pull/20662/commits/651ad3480a986e6e3bd41f8bfa0addb207f57335